/*
 Apply this patch to fetchpop1.9, such as 'patch < fetchpop1.9.patch2'
 This is fix for some POP3 servers that send CR/LF in reply

		seung-hong oh <oh@cis.ohio-state.edu> 
*/

*** fetchpop1.9/pop.c.orig	Tue Jun 25 21:28:06 1996
--- fetchpop1.9/pop.c	Sun Jun 30 22:41:12 1996
***************
*** 94,103 ****
  
  #ifndef LogOnlyNew
!   sprintf(buf,"There are %d new messages and total of %d messages.\n",newmsg,howmany);
    printmsg(buf,options->logging, options->daemon, logfd, 0, options->whichdeliver == 4);
    rpy = 0;
  #else
    if (!options->daemon)
!     printf("There are %d new messages and total of %d messages.\n",newmsg,howmany);
  #endif
    if ((options->remove == 2) && (last > 0)) {
--- 94,103 ----
  
  #ifndef LogOnlyNew
!   sprintf(buf,"You have %d new messages and total of %d messages.\n",newmsg,howmany);
    printmsg(buf,options->logging, options->daemon, logfd, 0, options->whichdeliver == 4);
    rpy = 0;
  #else
    if (!options->daemon)
!     printf("You have %d new messages and total of %d messages.\n",newmsg,howmany);
  #endif
    if ((options->remove == 2) && (last > 0)) {
***************
*** 153,157 ****
    int rpy;
    
!   if (!readsock(sockfd, argbuf, POPBUFLEN)) {
  #ifdef DEBUG
      printf("argbuf is %s\n", argbuf);
--- 153,157 ----
    int rpy;
    
!   if (!sockgetline(sockfd, argbuf, POPBUFLEN)) {
  #ifdef DEBUG
      printf("argbuf is %s\n", argbuf);
